client/cypress/integration/DashboardPage.spec.js

```diff
@@ -0,0 +1,98 @@
+/// <reference types="cypress" />
+
+describe('Erigolem Dashboard Page tests', () => {
+  before(() => {
+    cy.waitUntil(
+      () =>
+        cy
+          .request({
+            method: 'GET',
+            url: 'http://0.0.0.0:5000/getInstances',
+            headers: {
+              Authorization: 'Bearer 0x25bb4351B805884663d278F10A2096437Ee29855',
+            },
+          })
+          .its('status')
+          .then((status) => status === 200),
+      {
+        interval: 2000,
+        timeout: 60000,
+      },
+    );
+  });
+
+  beforeEach(() => cy.visit(''));
+
+  const newNodeLabel = ' is my new node';
+  const tabPanel = 'div[role=tabpanel]';
+
+  it('Displays Dashboard with no new nodes ', () => {
+    cy.contains("Ooops! It looks like you don't have any node currently running");
+  });
+
+  it('Checks if user can start New Node', () => {
+    cy.startNewNodeClick().then(() => {
+      cy.contains('Name your node');
+      cy.get('input[type=text]').type(newNodeLabel);
+
+      cy.contains('Choose network');
+      cy.get('label[for=network_rinkeby]').click();
+
+      cy.startNewNodeClick('Start node').then(() => cy.contains(newNodeLabel));
+    });
+  });
+
+  it('Checks if user can see his Active Nodes', () => {
+    cy.get(tabPanel).should('have.length.at.least', 1);
+  });
+
+  it('Checks if New Node is running', () => {
+    cy.waitUntil(
+      () =>
+        cy
+          .get(`span[role=status]`)
+          .invoke('text')
+          .then((text) => text === 'Running'),
+      {
+        interval: 5000,
+        timeout: 30000,
```

Comment:
I'm not sure how these timeouts work but when I tried running the tests on my machine this assertion timed out after 4 s:
```
1) Erigolem Dashboard Page tests
       Checks if New Node is running:
     AssertionError: Timed out retrying after 4000ms: Expected to find element: `span[role=status]`, but never found it.
      at cy.waitUntil.interval (http://localhost:8000/__cypress/tests?p=cypress/integration/DashboardPage.spec.js:132:27)
      at resolveValue (http://localhost:8000/__cypress/tests?p=cypress/support/index.js:206:20)
      at Context.waitUntil (http://localhost:8000/__cypress/tests?p=cypress/support/index.js:216:10)
```
I think we should wait up to 10 s.

Reply:
Package was removed and cypress config file changes default timeout to 30s, as it seems to be proper time to run node.